kselftests-production.yaml

```diff
@@ -1483,4 +1483,19 @@ projects:
     - kselftest-vsyscall-mode-native/net_so_txtime.sh
     url: https://bugs.linaro.org/show_bug.cgi?id=5331
     active: true
-    intermittent: false
+    intermittent: true
+  - environments: *environments_all
+    notes: >
+      selftests pidfd_pidfd_tests fails intermittently on x86_64, i386 and x15
+      selftests pidfd pidfd_test
+      pidfd: pidfd_test
+      [FAIL] 1 selftests pidfd pidfd_test TIMEOUT
+      selftests pidfd_pidfd_test [FAIL]
```

Comment:
Can you pinpoint where exactly this fails?

Reply:
The full test log shows TIMEOUT and when i run test case independently test getting PASS.

you can see the intermittent PASS/FAIL on x86_64 and i386.
https://qa-reports.linaro.org/lkft/linux-mainline-oe/tests/kselftest/pidfd_pidfd_test

On arm64 juno the failure log is,
# selftests pidfd pidfd_test
pidfd: pidfd_test_ #
# TAP version 13
version: 13_ #
# 1..4
: _ #
# # Parent pid 12275
Parent: pid_12275 #
# # Parent Waiting for Child (12276) to complete.
Parent: Waiting_for #
# # Time waited for child 0
Time: waited_for #
# Bail out! pidfd_poll check for premature notification on child thread exec test Failed
out!: pidfd_poll_check #
# # Planned tests != run tests (4 != 0)
Planned: tests_!= #
# # Pass 0 Fail 0 Xfail 0 Xpass 0 Skip 0 Error 0
Pass: 0_Fail #
[FAIL] 1 selftests pidfd pidfd_test # exit=1
selftests: pidfd_pidfd_test [FAIL]

https://qa-reports.linaro.org/lkft/linux-mainline-oe/build/v5.4-rc2-35-gd5001955c281/testrun/959063/log

ref:
https://lkft.validation.linaro.org/scheduler/job/978782#L14879